addons/reload/functions/fnc_getAmmoToLinkBelt.sqf

```diff
@@ -1,11 +1,11 @@
 #include "script_component.hpp"
 /*
  * Author: esteldunedain, phyma
- * Check if the target has an MG equiped with belt system that the player can link
+ * Check if the target has an MG equipped with belt system that a unit can link.
  *
  * Arguments:
- * 0: Player <OBJECT>
- * 1: Target <OBJECT>
+ * 0: Unit to execute the reload <OBJECT>
```

Comment:
_(EDIT: Changed to the wording in `fnc_startLinkingBelt.sqf`)_

```suggestion
 * 0: Unit executing the reload <OBJECT>
```

Reply:
The function is called before the actual reload, so I prefer the original wording.

Reply:
The problem is the original is confusing.

How about this ?
```suggestion
 * 0: Unit that would execute the reload <OBJECT>
```

Reply:
Or this ?
```suggestion
 * 0: Unit wanting to execute the reload <OBJECT>
```